### DISCUSSION TITLE
best codec for convert flac to wav

### DISCUSSION THREAD

**Discussion by fakiejohnny** [enzo1982/freac#452] [Audio Format Discussion]
Hello, I need wav files (for my Pioneer XDJ-700), but I only have flac files. What is the best encoder/the best setting to get a lossless wave file?

**Comment by enzo1982** [comment]
Hi, .wav files are usually lossless/uncompressed, so just select _"Windows Wave File Output"_ as the encoder in fre:ac.

If you changed any settings for that encoder before, you may want to check the settings and make sure _"auto select"_ is chosen for the _"Audio format"_ option in the encoder settings. It's the default, though, and you normally don't need to change it.

**Reply by fakiejohnny** [reply]
Thank you! I've already done that. I also selected "Signed 16 Bit PCM" but that should normally be identical to "automatic".
